server/api/crud/model_monitoring/deployment.py

```diff
@@ -117,34 +117,42 @@ def deploy_model_monitoring_stream_processing(
 
         :param stream_image:                The image of the model monitoring stream function.
                                             By default, the image is mlrun/mlrun.
+        :param overwrite:                   If true, overwrite the existing model monitoring stream. Default is False.
         """
 
-        self._check_if_already_deployed(
+        if not self._check_if_already_deployed(
             function_name=mm_constants.MonitoringFunctionNames.STREAM,
-        )
-        # Get parquet target value for model monitoring stream function
-        parquet_target = (
-            server.api.crud.model_monitoring.helpers.get_monitoring_parquet_path(
-                db_session=self.db_session, project=self.project
+            overwrite=overwrite,
+        ):
+            # Get parquet target value for model monitoring stream function
+            parquet_target = (
+                server.api.crud.model_monitoring.helpers.get_monitoring_parquet_path(
+                    db_session=self.db_session, project=self.project
+                )
+            )
+            if overwrite and stream_image.startswith("mlrun/mlrun"):
```

Comment:
As we talked, the image doesn't really matter. We just need to add the new trigger in this case.